metrics/network/cpu_stadistics_iperf.sh

```
@@ -0,0 +1,96 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'iperf3' network benchmark and
+# we retrieve cpu information while running the networking
+# test with 'perf'.
+
+set -e
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+TEST_NAME="cpu information"
+
+# Image name
+image="gabyct/network"
+# Client name
+client_name="client_iperf"
+# Server name
+server_name="server_iperf"
+# Target bandwidth
+bandwidth="1G"
+# Time in seconds to transmit
+timeout="160"
+# Sample period in seconds to measure
+sample_period="120"
+# File to save perf results
+TMP_FILE=$(mktemp cpuinfo.XXXXXXXXXX || true)
```

I don't think we want this to silently succeed so I'd drop the `|| true` bit.